database/migrations/2022_02_03_164059_increase_auth_id_length.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class IncreaseAuthIdLength extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('auth_id')->nullable()->change();
```

Comment:
Note to look out for: This could allow different auth_ids because they are no longer being cast from string to int...  but we will only know once this is in the wild.